src/utils/mediaManager.js

```
@@ -0,0 +1,416 @@
+		if (files.length > 1) {
+			duplicates.potentialDuplicates.push({
+				key,
+				files,
+				duplicateCount: files.length - 1,
+				wastedSpace: files[0].size * (files.length - 1)
+			});
+			duplicates.totalDuplicateSize += files[0].size * (files.length - 1);
+		}
+	}
+
+	delete duplicates.bySizeAndName; // Remove to reduce output size
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)